### DISCUSSION TITLE
2024/11/10/Hillbilly_Elegy_After_reading/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#31] [General]
# 2024/11/10/Hillbilly_Elegy_After_reading/

如题。《乡下人的悲歌》是特朗普副手J.D.万斯出版的一本自传，讲述了他在铁锈带的成长经历，或许对于理解当今世界变化格局，以及规划我们自身成长有一定的帮助。

https://wz.anoms.top/2024/11/10/Hillbilly_Elegy_After_reading/

**Comment by Hazelandinstant** [comment]
《布鲁克林有棵树》？听起来有些的相似，都是童年生活不够幸运，但是又因为有爱仿佛在糟糕生活中有了前进的光

**Reply by cyclinbox** [reply]
今天我读完了这本书，感觉很被故事情节温暖到——从主人公刚记事时写起，写父母亲的爱，写邻居和老师的爱，写主人公成长中的磕磕绊绊，一直到父亲因酗酒去世、自己半工半读着读完了大学（她真的好坚强！），并遇见所爱之人。想借助豆瓣上的一句书评来评价：  `「这是一本杰出的成长小说，纯真、希望、感动便是它带给我们的体验。生活？我们爱过、厌过甚至恨过，但终究不会忘怀。多年以后回首往事，如何诉说，已忘言」` 。

不过虽说都是“因为有爱仿佛在糟糕生活中有了前进的光”，《乡下人的悲歌》和《布鲁克林有棵树》的侧重点各有不同：《乡》的内核是荒凉落寞的，它更着重于强调主人公成长环境的糟糕（并且暗示一切原因都在糟糕的美国政客），以调动美国铁锈带选民的情绪，这也是为什么这本书在特朗普胜选后爆火；《布》的内核则是温暖和充满希望的，不论布鲁克林的环境如何艰苦，作者都在强调爱的力量，以及感恩与成长。这或许是两本书最大的不同。

总之感谢荐书！（也期待多多荐书~

**Reply by cyclinbox** [reply]
PS：最近突然意识到前段时间我可能又做错了一些事，也许一些话伤害到了你，实在是对不起😥向你表示抱歉……）
